examples/data_sources/traceable_service_id/data_source.tf

```diff
@@ -0,0 +1,8 @@
+data "traceable_service_id" "endpoint" {
+  service_name="nginx-automation-test"
```

Comment:
this is bit internal to our tests..use generic name s like test-service etc

Reply:
done